EstateReporting.IntegrationTests/Common/Contract.cs

```diff
@@ -0,0 +1,100 @@
+﻿namespace EstateReporting.IntegrationTests.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public class Contract
+    {
+        #region Properties
+
+        /// <summary>
+        /// Gets or sets the contract identifier.
+        /// </summary>
+        /// <value>
+        /// The contract identifier.
+        /// </value>
+        public Guid ContractId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the description.
+        /// </summary>
+        /// <value>
+        /// The description.
+        /// </value>
+        public String Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the operator identifier.
+        /// </summary>
+        /// <value>
+        /// The operator identifier.
+        /// </value>
+        public Guid OperatorId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the products.
+        /// </summary>
+        /// <value>
+        /// The products.
+        /// </value>
+        public List<Product> Products { get; set; }
+
+        #endregion
+
+        #region Methods
+
+        /// <summary>
+        /// Adds the product.
+        /// </summary>
+        /// <param name="productId">The product identifier.</param>
+        /// <param name="name">The name.</param>
+        /// <param name="displayText">The display text.</param>
+        /// <param name="value">The value.</param>
+        public void AddProduct(Guid productId,
+                               String name,
+                               String displayText,
+                               Decimal? value = null)
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6339123)